build-aux/toolchain/build/dependencies.sh

```diff
@@ -0,0 +1,49 @@
+# Check if all dependencies are installed
+CROSSTOOLCHAIN_DEPENDENCIES_TOOL="make wget tar xz gcc g++ "
+CROSSTOOLCHAIN_DEPENDENCIES_LIBS=""
+
+# Add gcc dependencies
+CROSSTOOLCHAIN_DEPENDENCIES_TOOL+="bison flex texindex "
+CROSSTOOLCHAIN_DEPENDENCIES_LIBS+="libgmp libmpc libmpfr "
+
+# Add llvm (optional) dependencies
+if [[ "$COMPILER" == "llvm" ]]; then
+    CROSSTOOLCHAIN_DEPENDENCIES_TOOL+="cmake python3 pip3"
+    CROSSTOOLCHAIN_DEPENDENCIES_LIBS+="libz"
+fi
+
+# Test if all required dependencies are installed
+echo "[i] - Checking dependencies..."
+
+# Check if all commands are installed
+for name in $CROSSTOOLCHAIN_DEPENDENCIES_TOOL;
+do
+    if [[ ! $(which "$name" 2>/dev/null) ]]; then
+        echo "[!] - $name is not installed.";
+        deps='true';
+    fi
+done
+
+# Check befor
+if [[ ! $(which ldconfig 2>/dev/null) ]]; then 
```

Comment:
I'd rather have each `if/then` statement block in the scripts with the `then` on a new line, except if the `if x; then y; else z; fi` fits on a single 80-character line.

Reply:
done